operations/firmware/Updating_Firmware_m001.md

```diff
@@ -0,0 +1,85 @@
+## Updating Firmware / BIOS for NCN m001
+
+**Run on m001**
+
+**Prerequisite:**
+* IP Address of m001 bmc
+* IP Address of m001
+* Root password for m001 bmc
+
+### Find the Model Name
+
+***On HPE Nodes***
+
+`m001# curl -k -u root:password https://ipaddressOfBMC/redfish/v1/Systems/1 | jq .Model`
+
+***On Gigbyte Nodes***
+
+`m001# curl -k -u root:password https://ipaddressOfBMC/redfish/v1/Systems/Self | jq .Model`
+
+### Get the Firmware Images
+
+`m001# cray fas images list --format json | jq .[] | .[] | select(.models | index("ModelName"))`

```

Comment:
Update section to improve workflow:

---

1. View a list of images store in FAS that are ready to be flashed.
   
   Replace "ModelName" in the following command with the value retrieved in the previous section.
   
   ```
   ncn-m001# cray fas images list --format json | jq .[] | .[] | select(.models | index("ModelName"))
   ```
   
   Locate the image in the returned output that is required to flash ncn-m001 firmware and/or BIOS.
   
   Look for the returned s3URL. For example:

   ```
   "s3URL": "s3:/fw-update/4e5f569a603311eb96b582a8e219a16d/image.RBU"
   ```

1. Get the firmware images using the s3URL path from the previous step.
   
   ```
   ncn-m001# cray artifacts get fw-update 4e5f569a603311eb96b582a8e219a16d/image.RBU image.RBU
   ```

   `4e5f569a603311eb96b582a8e219a16d/image.RBU` is the path in the s3URL. `image.RBU` is the name of the file to save the image on local disk.